sdk/Lusid.Sdk.Tests/tutorials/Valuations/InlineBondValuation.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using Lusid.Sdk.Api;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Utilities;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Valuations
+{
+    [TestFixture]
+    public class InLineBondValuation
+    {
+        private ILusidApiFactory _apiFactory;
+
+        // This defines the scope that entities will be created in
+        private const string TutorialScope = "Testdemo";
+
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            _apiFactory = LusidApiFactoryBuilder.Build("secrets.json");
+        }
+        
+        [Test]
+        public void Run_BondInLineValuation()
+        {
+            // Define a flow convention for the Bond
+            // In this instance, the bond will pay a GBP coupon every 6 months
+            // Accrued interest will be calculated on the ACT/365 day count convention
+            
+            var flowConventions = new FlowConventions(
+                scope: null,
+                code: null,
+                currency: "GBP",
+                paymentFrequency: "6M",
+                rollConvention: FlowConventions.RollConventionEnum.MF,
+                dayCountConvention: FlowConventions.DayCountConventionEnum.Act365,
+                holidayCalendars: new List<string>(),
+                settleDays: 2,
+                resetDays: 2
+            );
+            
+            // Use the GetValuationOfWeightedInstruments to price our Bond
+            // We price the Bond for a date between the Bond's startDate and maturityDate
+
+            ListAggregationResponse bondValuationResult = _apiFactory.Api<IAggregationApi>().GetValuationOfWeightedInstruments(
```

Comment:
where are the quotes coming from?

Reply:
No quotes needed, right?
The `Bond()` model has all the economic details needed to calculate the PV.

Reply:
Where is the (discount) curve coming from?

Reply:
Added that now. See the `UpsertRateCurves` method.